pages/_app.tsx

```diff
@@ -15,14 +14,7 @@ function MyApp({ Component, pageProps }: AppProps): React.ReactElement {
   const { locale = "en" } = useRouter();
 
   const messages = useMemo(() => {
-    switch (locale) {
-      case "en":
-        return English;
-      case "fr":
-        return French;
-      default:
-        return English;
-    }
+    return locale === "fr" ? French : undefined;
```

Comment:
I'm unsure here why do we try to return undefined here instead of English ?

Reply:
It defaults to the English text when the value is null. We don't need to import the `en.json` file.